### PR TITLE
Odometer permission to both Admin or Manager

### PR DESCRIPTION
**Pull request by parveenkumardeeva**

### CODE REVIEW

**Comment by tananaev** [comment]
This doesn't seem correct. It will allow any user to do it.

**Comment by parveenkumardeeva** [comment]
Oh Sorry, Got the error.

File: src/org/traccar/database/PermissionsManager.java [hunk 1]
```diff
@@ -300,7 +300,13 @@ public void checkGroup(long userId, long groupId) throws SecurityException {
     }
 
     public void checkDevice(long userId, long deviceId) throws SecurityException {
-        if (!Context.getDeviceManager().getUserItems(userId).contains(deviceId) && !getUserAdmin(userId)) {
+        if (!Context.getDeviceManager().getUserItems(userId).contains(deviceId)) {
+            checkDeviceManagerOrAdmin(userId, deviceId);
+        }
+    }
+
+    public void checkDeviceManagerOrAdmin(long userId, long deviceId) throws SecurityException {
```
Comment (tananaev) [inline src/org/traccar/database/PermissionsManager.java]: I don't think we need a separate method. Please implement it the same way it's implement in other places.

Comment (parveenkumardeeva) [reply src/org/traccar/database/PermissionsManager.java]: Then We have to implement whole logic in the Device Resource's updateTotalDistance method or change the checkDeviceManagerOrAdmin method name?

Comment (tananaev) [reply src/org/traccar/database/PermissionsManager.java]: You don't need another method and you don't need to move the logic anywhere. Just look at the `BaseObjectResource` example.

**Comment by parveenkumardeeva** [comment]
Dear Anton,

I don't find any better way to do this. Please add this feature yourself if you still don't found the OK code.

**Comment by tananaev** [comment]
Looks good now. Merged, thanks.

**Pull request merged**